vcr/api/v2/holder_test.go

```diff
@@ -40,7 +40,7 @@ var organizationQuery = `
 {
 	"query": {
 		"@context": ["https://www.w3.org/2018/credentials/v1","https://nuts.nl/credentials/v1"],
-		"type": ["VerifiableCredential", "NutsOrganizationCredential"],
+		"type": "NutsOrganizationCredential",
```

Comment:
This would be an incorrect query. Without `VerifiableCredential` as type, the default VC fields will not be used in the query.